docs/reference/query-dsl/geo-shape-query.asciidoc

```diff
@@ -4,9 +4,9 @@
 <titleabbrev>Geo-shape</titleabbrev>
 ++++
 
-Filter documents indexed using the `geo_shape` type.
+Filter documents indexed using the `geo_shape` or `geo_point` type.
```

Comment:
I think we will want to add a section explicitly listing which shape types are not supported for `geo_point` types:  
* POINT
* LINE
* MULTIPOINT
* MULTILINE

And I think we can eventually add these as new `LatLonPoint` queries in Lucene by reusing our `relate` methods? Which would give us full parity. /cc @iverase

Reply:
Added to Documentation